tuxemon/core/components/event/actions/player.py

```diff
@@ -358,3 +358,24 @@ def player_resume(self, game, action):
             return
 
         world.menu_blocking = False
+
+    def remove_monster(self, game, action):
+        """Removes a monster to the current player's party if the monster is there.
+
+        :param game: The main game object that contains all the game's variables.
+        :param action: The action (tuple) retrieved from the database that contains the action's
+            parameters
+
+        :type game: core.control.Control
+        :type action: Tuple
+
+        :rtype: None
+        :returns: None
+
+        Valid Parameters: monster_slug
+        """
+        monster_slug = action.parameters[0]
+
+        monster = game.player1.find_monster(monster_slug)
+        if monster:
+                game.player1.remove_monster(monster)
```

Comment:
This line is indented too much, but other than that everything looks great! Thanks for the PR!